.gitattributes

```diff
@@ -0,0 +1 @@
+packages/jest-config/src/__tests__/jest-preset.json eol=lf
```

Comment:
can we fix the test instead?

Reply:
To do this I guess I need to replace CRLF with LF in runtime here (https://github.com/facebook/jest/blob/master/packages/jest-config/src/__tests__/normalize.test.js#L936-L950):

```javascript
    jest.doMock('/node_modules/react-native/jest-preset.json', () =>
      require.requireActual('./jest-preset.json'),
    );
```

I'm not sure how to do this properly and it doesn't seem like a good solution to me. Here is the failing output for the details:

```
Click to update snapshot for 'throws when preset is invalid' test

Error: expect(value).toMatchSnapshot()

Received value does not match stored snapshot "preset throws when preset is invalid 1".

- Snapshot
+ Received

  "<red><bold><bold>● <bold>Validation Error</>:</>
  <red></>
  <red>  Preset <bold>react-native</> is invalid:</>
- <red>  Unexpected token } in JSON at position 104</>
+ <red>  Unexpected token } in JSON at position 110</>
  <red></>
  <red>  <bold>Configuration Documentation:</></>
  <red>  https://facebook.github.io/jest/docs/configuration.html</>
  <red></>"
    at Object.test (E:\src\jest\packages\jest-config\src\__tests__\normalize.test.js:949:8)
    at Object.asyncFn (E:\src\jest\packages\jest-jasmine2\build\jasmine_async.js:108:37)
    at resolve (E:\src\jest\packages\jest-jasmine2\build\queue_runner.js:56:12)
    at new Promise (<anonymous>)
    at mapper (E:\src\jest\packages\jest-jasmine2\build\queue_runner.js:43:19)
    at promise.then (E:\src\jest\packages\jest-jasmine2\build\queue_runner.js:87:41)
    at <anonymous>
```